### PR TITLE
Add package utility

### PR DESCRIPTION
**Pull request by anthonybishopric**
Standard rake task to create builds to use in tagged versions of P2

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'json'
```
Comment (petertseng) [inline Rakefile]: Is there actually any jsonning being done here?

**Comment by petertseng** [comment]
Other than the json thing I'm okay with this in general. I did `rake build` and got a `builds/darwin/` dir with reasonable contents (`p2-0.0.3-40-gecd5ef8-darwin`, `p2-0.0.3-40-gecd5ef8-darwin.tar.gz` and `p2.tar.gz`)

**Comment by petertseng** [comment]
The tar has the `p2-*` binaries under a `./bin` dir, seems reasonable

File: Rakefile (hunk 1 above)
Comment (anthonybishopric) [inline Rakefile]: No, there isn't - will remove

**Pull request merged**